src/sql/workbench/contrib/notebook/browser/notebook.component.ts

```diff
@@ -146,46 +146,48 @@ export class NotebookComponent extends AngularDisposable implements OnInit, OnDe
 			// For Escape - the focused element is the div.notebook-preview or textarea.inputarea of the cell, so we need to make sure that it is a descendant of the current active cell
 			//  on the current active editor.
 			const activeCellElement = this.container.nativeElement.querySelector(`.editor-group-container.active .notebook-cell.active`);
+			const findWidgetVisible = document.querySelector(`.editor-widget.find-widget.visible`);
```

Comment:
It looks like there's a `isFindActive` on CellView as well - would it be possible to use that instead?